test/jdk/jdk/internal/loader/NativeLibraries/Main.java

```diff
@@ -55,7 +57,9 @@ public static void main(String... args) throws Exception {
         test.unload();
 
         // load zip library from JDK
-        test.load(System.mapLibraryName("zip"), true /* succeed */);
+        if (!Platform.isStatic()) {
+            test.load(System.mapLibraryName("zip"), true /* succeed */);
```

Comment:
Hi, the other tests which also call `NativeLibrariesTest.load(String, boolean)`, does the other tests also need this check `!Platform.isStatic()`